cloud/models/match.js

```diff
@@ -6,6 +6,12 @@ var Match = Parse.Object.extend("Match", {
   isCurrent: function(){
     return this.get('endedAt') == null;
   },
+  forfeit: function(){
+    return this.save({
+      winner: 'none',
```

Comment:
We need to make sure that the client can handle this value, since I remember assuming at one point that winner is either A or B.